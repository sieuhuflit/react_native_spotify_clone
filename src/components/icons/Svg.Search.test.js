import React from 'react';
import { shallow } from 'enzyme';
import SvgSearch from './Svg.Search';

const setup = propOverrides => {
  const props = Object.assign({}, propOverrides);
  const wrapper = shallow(<SvgSearch {...props} />);
  return {
    props,
    wrapper
  };
};

describe('SvgSearch check snapshot', () => {
  test('should match snapshot', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
