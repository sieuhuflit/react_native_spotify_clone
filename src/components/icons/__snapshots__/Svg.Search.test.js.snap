// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgSearch check snapshot should match snapshot 1`] = `ShallowWrapper {}`;
